### PR TITLE
Promote to prod

### PR DESCRIPTION
**Pull request by NabaSadiaSiddiqui**

### CODE REVIEW

**Review by ashaduzzamane — COMMENTED** [review]
Change the image file name into something more descriptive

File: src/Components/Dashboard/Cards/CommunityCard.js [hunk 1]
```diff
@@ -808,6 +901,51 @@ class JobApplicationCard extends Component {
...
         </Dialog>
+
+        <Dialog
+        open={this.state.showConnectDialog}
+        onClose={this.closeConnectDialog}
+        aria-labelledby="alert-dialog-create"
+        aria-describedby="alert-dialog-description-create"
+        >
+          <DialogTitle id="alert-dialog-create">
+            {"Accept Connection?"}
```
Comment (tahbibO) [inline src/Components/Dashboard/Cards/CommunityCard.js:912]: This should be Send Request

File: README.md [hunk 2]
```diff
@@ -1,66 +1,82 @@
-## Aspire Frontend
+# aspire-Frontend
```
Comment (tahbibO) [inline README.md:1]: fix small grammar and spelling errors

**Review by tahbibO — CHANGES_REQUESTED** [review]
- update readme to fix mistakes
- change connection request header to "send connection"

**Review by tahbibO — APPROVED** [review]

**Review by tahbibO — APPROVED** [review]

**Pull request merged**